hubblestack/extmods/fileserver/s3fs.py

```diff
@@ -354,8 +355,11 @@ def _init():
     specified and cache the data to disk.
     """
     cache_file = _get_buckets_cache_filename()
-    exp = time.time() - S3_CACHE_EXPIRE
+    s3_key_kwargs = _get_s3_key()
+    cache_expire_time = s3_key_kwargs['cache_expire'] if s3_key_kwargs.get('cache_expire', None) else S3_CACHE_EXPIRE
+    exp = time.time() - float(cache_expire_time)
 
+    log.info('S3 cache expire time is {}s'.format(cache_expire_time))
```

Comment:
Shouldn't this be something like `log.info('S3 cache expire time is %ds', cache_expire_time)` to avoid `logging-format-interpolation`?